src/mceditlib/anvil/adapter.py

```diff
@@ -357,10 +357,22 @@ def __init__(self, metadataTag):
     SizeOnDisk = nbtattr.NBTAttr('SizeOnDisk', nbt.TAG_Long, 0)
     RandomSeed = nbtattr.NBTAttr('RandomSeed', nbt.TAG_Long, 0)
     Time = nbtattr.NBTAttr('Time', nbt.TAG_Long, 0)  # Age of the world in ticks. 20 ticks per second; 24000 ticks per day.
+    DayTime = nbtattr.NBTAttr('DayTime', nbt.TAG_Long, 0)  # Amount of ticks since Day 1, 6:00
     LastPlayed = nbtattr.NBTAttr('LastPlayed', nbt.TAG_Long, time.time() * 1000)
-
+    Difficulty = nbtattr.NBTAttr('Difficulty', nbt.TAG_Byte, 0)
     LevelName = nbtattr.NBTAttr('LevelName', nbt.TAG_String, "Untitled World")
+    hardcore = nbtattr.NBTAttr('hardcore', nbt.TAG_Byte, False)
+    allowCommands = nbtattr.NBTAttr('allowCommands', nbt.TAG_Byte, False)
+    DifficultyLocked = nbtattr.NBTAttr('DifficultyLocked', nbt.TAG_Byte, False)
+
+
+    SpawnX = nbtattr.NBTAttr('SpawnX', nbt.TAG_Int, 0)
```

Comment:
I just found one already (adapter.py, line 382: 
    def worldSpawnPosition(self):
        return Vector(*[self.rootTag[i].value for i in ("SpawnX", "SpawnY", "SpawnZ")])